app/models/manageiq/providers/openstack/inventory/collector.rb

```diff
@@ -46,7 +46,7 @@ def initialize_inventory_sources
     @host_aggregates           = []
     @key_pairs                 = []
     @images                    = []
-    @orchestration_stacks      = []
+    @orchestration_stacks      = nil
```

Comment:
@mansam I wonder if we can just ditch these definitions and use the nil checks everywhere?

Reply:
Yeah, these can probably be changed in the same way.